src/index.json

```diff
@@ -59042,6 +59042,14 @@
                     "version": "2.26.1"
                 },
                 "sha256Digest": "caac6f58f9f181cb94e4673f0e0affa10d8986ac84b30b6aa547a2d28901090e"
+            },
+            {
+                "downloadUrl": "https://azuremlsdktestpypi.blob.core.windows.net/wheels/sdk-cli-v2-public/ml-2.27.0-py3-none-any.whl",
```

Comment:
I notice this object is much shorter compared to pervious versions. Is that intentional?

Reply:
fixed